clients/java/src/main/java/io/camunda/client/impl/search/query/VariableQueryImpl.java

```diff
@@ -65,8 +66,8 @@ public FinalSearchQueryStep<Variable> requestTimeout(final Duration requestTimeo
   }
 
   @Override
-  public CamundaFuture<SearchQueryResponse<Variable>> send() {
-    final HttpCamundaFuture<SearchQueryResponse<Variable>> result = new HttpCamundaFuture<>();
+  public ZeebeFuture<SearchQueryResponse<Variable>> send() {
+    final HttpZeebeFuture<SearchQueryResponse<Variable>> result = new HttpZeebeFuture<>();
```

Comment:
🔧 We need to account for the latest renamings in the Java client to `CamundaXyz` classes in this PR.